packages/api/src/beacon/routes/lightclient.ts

```diff
@@ -19,6 +22,17 @@ export type LightClientBootstrap = {
   currentSyncCommitteeBranch: Uint8Array[];
 };
 
+export type VersionedLightClientUpdate = {
+  version: ForkName;
+  data: altair.LightClientUpdate;
+};
+
+export type VersionedLightClientOptimisticUpdate = {version: ForkName; data: altair.LightClientOptimisticUpdate};
+
+export type VersionedLightClientFinalityUpdate = {version: ForkName; data: altair.LightClientFinalityUpdate};
+
+export type VersionedLightClientBootstrap = {version: ForkName; data: altair.LightClientBootstrap};
```

Comment:
can we not declare new versioned types and just include version, that way the real data type's not obscured as defining versioned types add little of the reuse value, else looks good

Reply:
yeah just add version field below